cmd/baton-calendly/main.go

```diff
@@ -5,40 +5,49 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/conductorone/baton-sdk/pkg/cli"
+	"github.com/conductorone/baton-calendly/pkg/connector"
+	configSchema "github.com/conductorone/baton-sdk/pkg/config"
 	"github.com/conductorone/baton-sdk/pkg/connectorbuilder"
+	"github.com/conductorone/baton-sdk/pkg/field"
 	"github.com/conductorone/baton-sdk/pkg/types"
 	"github.com/grpc-ecosystem/go-grpc-middleware/logging/zap/ctxzap"
+	"github.com/spf13/viper"
 	"go.uber.org/zap"
+)
 
-	"github.com/conductorone/baton-calendly/pkg/connector"
+const (
+	version       = "dev"
+	connectorName = "baton-calendly"
 )
 
-var version = "dev"
+var (
+	Token               = field.StringField(connector.Token, field.WithRequired(true), field.WithDescription("Personal Access Token used to authenticate with the Calendly API."))
+	configurationFields = []field.SchemaField{Token}
+	ctx                 = context.Background()
```

Comment:
I don't know a enough Go, but I think the `context.Background()` call needs to be in the `main()` function.

Reply:
Yes, indeed. We do not need a global context within the main package. I'll fix it

Reply:
Fixed